src/styling/MovieDetails.css

```diff
@@ -0,0 +1,140 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  color: white;
+  font-family: "Montserrat";
+}
+
+.details-container {
+  background-size: cover;
+  background-position: center center;
+  height: 100vh;
+  padding: 10px 16px 0 16px;
+  margin: 0;
+  overflow-x: hidden;
+  background-attachment: fixed;
+}
+
+.details-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.movie-image {
+  width: 200px;
+  border: solid 5px white;
+  box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
+  margin-bottom: 20px;
+}
+
+.movie-text {
+  text-shadow: 1px 1px #4d4d4d;
+  background-color: rgb(0, 0, 0, 0.5);
+  background-position: center;
+  padding: 10px;
+  margin-bottom: 10px;
+  border-radius: 8px;
+}
+
+.movie-text p {
+  margin-bottom: 0;
+}
+
+.movie-title {
+  font-weight: 700;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.original-title {
+  font-size: 16px;
+  padding-top: 4px;
+  margin: 0;
+}
+
+.movie-ranking {
+  font-weight: 600;
+  text-align: right;
+  margin: 0;
+  padding: 0;
+}
+
+.genre-list {
+  display: flex;
+  margin-top: 0;
+  padding: 0;
+  justify-content: left;
+  list-style-type: none;
+  flex-wrap: wrap;
+}
+
+.genre-list li {
+  padding: 0px 10px 10px 0;
+  font-weight: 700;
+}
+
+.back-link {
```

Comment:
Maybe put the back-to-movie link aligned with the rest of the content? On mobile and tablet. 

![Screenshot 2024-04-08 at 20 28 15](https://github.com/Technigo/project-movies-vite/assets/152007424/6155a7f0-e07c-437a-94f0-99231c5115a3)